process/prefixer.go

```diff
@@ -32,19 +34,40 @@ func (p *Prefixer) Write(data []byte) (n int, err error) {
 	offset := 0
 	out := make([]byte, 0, len(data))
 
-	// loop through newlines
-	for offset < len(data) {
-		next := bytes.IndexRune(data[offset:], '\n')
+	// loop through line breaks and add prefix
+	for l := len(data); offset < l; {
```

Comment:
Would a [`bufio.Scanner`](https://golang.org/pkg/bufio/#Scanner) be appropriate here, with a custom split function detecting new line or escape?

Reply:
> Programs that need more control over error handling or large tokens, or must run sequential scans on a reader, should use bufio.Reader instead.

Reply:
bufio.Scanner fails on long lines, it's a pain.